src/macros.rs

```diff
@@ -10,6 +10,7 @@ macro_rules! T {
     (rec)     => ($crate::SyntaxKind::TOKEN_REC);
     (then)    => ($crate::SyntaxKind::TOKEN_THEN);
     (with)    => ($crate::SyntaxKind::TOKEN_WITH);
+    (ident)   => ($crate::SyntaxKind::TOKEN_IDENT);
```

Comment:
`ident` is not a literal keyword. This is confusing.
Just using `TOKEN_IDENT` when needed should be fine.

```suggestion
```

Reply:
It doesn't matter that `ident` is not a literal keyword, as the macro just gets tokens. I don't think it is that confusing. A note can be put above the macro if needed